apps/essence/server/services/db/repository/menu.ts

```
@@ -37,6 +37,39 @@ export class Menu {
+  static async findCategoryAndProductConnection(categoryId: string, productId: string) {
+    return useDatabase().query.productsInMenuCategories.findFirst({
+      where: (connections, { eq, and }) =>
+        and(
+          eq(connections.menuCategoryId, categoryId),
+          eq(connections.productId, productId),
+        ),
+    })
+  }
+
+  static async attachProductToCategory(categoryId: string, productId: string) {
+    const connection = await this.findCategoryAndProductConnection(categoryId, productId)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)